target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -173,9 +173,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch0" \
 			"1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "0:wan" "6@eth0"
 		;;
-	lava,lr-25g001)
+	lava,lr-25g001|\
+	telewell,tw-4g-lte)
 		ucidef_add_switch "switch0" \
-			"1:lan" "2:lan" "3:lan" "4:lan" "5:wan" "0@eth0"
+			"1:lan:4" "2:lan:3" "3:lan:2" "4:lan:1" "5:wan" "0@eth0"
```

Comment:
Separate device, separate change.

Reply:
Devices are now separated. Only jboot generic stuff is piled as one. Hope this is finaly ok for everybody.
Should I make pr to fix Lava router leds config or does someone else do it?

Reply:
If you resolve the question, it's hard for people to answer it.

But yes, the split was what I asked for regarding network config.